kart/tabular/working_copy/base.py

```diff
@@ -733,6 +742,21 @@ def get_dirty_pks(self, dataset):
             )
             return (row[0] for row in r)
 
+    def get_feature(self, dataset, pk, allow_schema_diff=True):
+        """Gets the features for the given dataset."""
+        meta_diff = self.diff_dataset_to_working_copy_meta(dataset)
+        if not allow_schema_diff and meta_diff.recursive_get(["meta", "schema.json"]):
```

Comment:
`if not x and y` might _technically_ have a well-defined meaning but it's hard to mentally parse. Add some parentheses

Reply:
Done